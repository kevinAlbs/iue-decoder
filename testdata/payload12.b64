DIkAAAAFZAAgAAAAAPGmZcUzdE/FPILvRSyAScGvZparGI2y9rJ/vSBxgCujBXMAIAAAAACi1RjmndKqgnXy7xb22RzUbnZl1sOZRXPOC0KcJkAxmQVsACAAAAAApJtKPW4+o9B7gAynNLL26jtlB4+hq5TXResijcYet8USY20AAAAAAAAAAAAA