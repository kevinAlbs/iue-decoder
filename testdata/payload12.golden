BlobSubtype=12 (FLE2FindEqualityPayloadV2)
EDCDerivedFromDataToken=F1A665C533744FC53C82EF452C8049C1AF6696AB188DB2F6B27FBD2071802BA3 ("d":{"$binary":{"base64":"8aZlxTN0T8U8gu9FLIBJwa9mlqsYjbL2sn+9IHGAK6M=","subType":"00"}})
ESCDerivedFromDataToken=A2D518E69DD2AA8275F2EF16F6D91CD46E7665D6C3994573CE0B429C26403199 ("s":{"$binary":{"base64":"otUY5p3SqoJ18u8W9tkc1G52ZdbDmUVzzgtCnCZAMZk=","subType":"00"}})
ServerDerivedFromDataToken=A49B4A3D6E3EA3D07B800CA734B2F6EA3B65078FA1AB94D745EB228DC61EB7C5 ("l":{"$binary":{"base64":"pJtKPW4+o9B7gAynNLL26jtlB4+hq5TXResijcYet8U=","subType":"00"}})
MaxContentionCounter=0 ("cm":0)
